packages/facebook/facebook_server.js

```diff
@@ -50,16 +50,32 @@ var getTokenResponse = function (query) {
 
   var responseContent;
   try {
-    // Request an access token
-    responseContent = HTTP.get(
-      "https://graph.facebook.com/v2.2/oauth/access_token", {
-        params: {
-          client_id: config.appId,
-          redirect_uri: OAuth._redirectUri('facebook', config),
-          client_secret: OAuth.openSecret(config.secret),
-          code: query.code
-        }
-      }).content;
+    // Request an access token, or request access token info if it was already given
+    // DDP clients should make a post request sending the accessToken before logging in with oauth credentialtoken/secret
```

Comment:
What is the post request? Is that the first `HTTP.get` to `/app` below, or something additional that needs to be done that's not here?